data-prepper-plugins/otel-metrics-raw-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/otelmetrics/OTelMetricsRawProcessor.java

```diff
@@ -216,7 +219,10 @@ private List<? extends Record<? extends Metric>> mapHistogram(final io.opentelem
                     if (otelMetricsRawProcessorConfig.getCalculateHistogramBuckets()) {
                         builder.withBuckets(OTelProtoCodec.createBuckets(dp.getBucketCountsList(), dp.getExplicitBoundsList()));
                     }
-                    return builder.build();
+                    System.out.println("......FLATTEN..."+flattenAttributesFlag);
```

Comment:
This should probably be removed, also the other ones in this class

Reply:
Good catch. Yes, please remove all the `System.out` calls @kkondaka .

Reply:
This was included in #2176.